extensions.tf

```diff
@@ -73,6 +38,25 @@ resource "azurerm_subscription_policy_assignment" "vm" {
   ]
 }
 
+# Enabling Role for Vulnerability assessment
+data "azurerm_role_definition" "security_admin" {
+  name = "Security Admin"
+}
+
+resource "azurerm_role_assignment" "va_auto_provisioning_identity_role" {
+  count = contains(var.mdc_plans_list, "VirtualMachines") ? 1 : 0
+
+  principal_id       = azurerm_subscription_policy_assignment.vm["mdc-va-autoprovisioning"].identity[0].principal_id
+  scope              = data.azurerm_subscription.current.id
+  role_definition_id = data.azurerm_role_definition.security_admin.id
+
+  depends_on = [
+    azurerm_subscription_policy_assignment.vm["mdc-va-autoprovisioning"],
```

Comment:
Since we've referred `azurerm_subscription_policy_assignment.vm["mdc-va-autoprovisioning"]` in line 49 Terraform knew the dependency between these two resources, so we can just remove this explicity `depends_on` item.

Reply:
Removed it, thanks!